build/settings.xml

```diff
@@ -0,0 +1,346 @@
+<settings>
```

Comment:
I don't think this file should be in the pull request

Reply:
Sure, I'll delete it.

Reply:
Done.

Reply:
I've just signed the ECA as user lauritziu_ii@yahoo.de. Is that enough. Do you need anything else?